pwndbg/commands/__init__.py

```diff
@@ -238,6 +238,14 @@ def _OnlyWhenHeapIsInitialized(*a, **kw):
             print("%s: Heap is not initialized yet." % function.__name__)
     return _OnlyWhenHeapIsInitialized
 
+def OnlyWithLibcDebugSyms(function):
+    @functools.wraps(function)
+    def _OnlyWithLibcDebugSyms(*a, **kw):
+        if pwndbg.heap.current.libc_has_debug_syms():
+            return function(*a, **kw)
+        else:
+            print("%s: This command only works with libc debug symbols." % function.__name__)
```

Comment:
We probably should be a bit more verbose/explicit here: e.g. what people can do in this case?

1) we could inform them about some packages they could install - however this is very distro-specific
2) we could create a markdown page describing this stuff (e.g. what to do on a given distro or how to add debug symbols to your libc?)

We potentially also want to display the libc that is being loaded in the binary - at least I think that would be awesome here. @mzr do you think it is possible?

Reply:
I think we could link to [gdb docs](https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html). IMO it's well explained there.

When it comes to automatic identification of libc version, we can use 
`curl -L -X GET 'https://libc.blukat.me?q=__libc_start_main_ret:e81,_IO_2_1_stdin_:5c0'` and extract information from response.

Reply:
Let's extend it as:
```
%s: This command only works with libc debug symbols.
They can probably be installed via the package manager of your choice.
See also: https://sourceware.org/gdb/onlinedocs/gdb/Separate-Debug-Files.html
```

Reply:
Done.